zookeeper-server/src/test/java/org/apache/zookeeper/server/backup/RestorationToolTest.java

```diff
@@ -384,4 +387,151 @@ public void testRestoreToTimestampByCommandLine() throws IOException {
     Assert.assertTrue(restoreTool.runWithRetries(cl));
     validateRestoreCoverage(txnCnt);
   }
+
+  @Test
+  public void testSpotRestorationTool() throws IOException, InterruptedException, KeeperException {
+    //TEST 1.
+    //Target node: /testsr
+    // This test is going to test three scenarios:
+    //    1. Nodes exist in backup files but not in the running zk server
+    //        Path => value:
+    //          /testsr/restore => restore
+    //          /testsr/restore/node0 => restore0
+    //        Expected: nodes are created in the running zk server
+    //    2. Nodes not exist in backup files but in the running zk server
+    //        Path => value:
+    //          /testsr/deprecated => deprecated
+    //          /testsr/deprecated/node0 => deprecated0
```

Comment:
This "deprecated" terminology is confusing - could we find a different term to describe this?